src/shared/netif-naming-scheme.h

```diff
@@ -32,6 +32,7 @@ typedef enum NamingSchemeFlags {
         NAMING_LABEL_NOPREFIX      = 1 << 7, /* Don't prepend ID_NET_LABEL_ONBOARD with interface type prefix */
         NAMING_NSPAWN_LONG_HASH    = 1 << 8, /* Shorten nspawn interfaces by including 24bit hash, instead of simple truncation  */
         NAMING_BRIDGE_NO_SLOT      = 1 << 9, /* Don't use PCI hotplug slot information if the corresponding device is a PCI bridge */
+        NAMING_SLOT_NONZERO        = 1 << 10, /* Ignore PCI hotplug slots with leading zeros */
```

Comment:
Maybe `NAMING_NO_SLOT_ZERO_PADDING` or so? `NONZERO` sounds slightly ambiguous for me.

Reply:
`NAMING_SLOT_NO_LEADING_ZERO` maybe